docs/config.rst

```diff
@@ -192,7 +192,7 @@ Features
    :type: string
    :value: ""
 
-   Delimiter to shorten the Kubernetes context.
+   Delimiter to shorten the Kubernetes context by removing a suffix.
```

Comment:
The changes on this file duplicate the changes you added in 545d297, which conflict.

Reply:
Ugh - merge resolution goofs. I'll take care of this when I rebase to pull in your branches (looking at those next).

Reply:
I believe this is resolved as of today's rebase.